package.json

```diff
@@ -41,25 +41,26 @@
     "body-parser": "1.19.0",
     "dotenv": "4.0.0",
     "express": "4.17.1",
-    "node-lambda": "0.16.0",
+    "node-lambda": "0.17.0",
     "pg": "^8.0.3",
     "promise": "8.1.0",
     "underscore": "1.9.1",
     "winston": "3.2.1"
   },
   "devDependencies": {
     "@babel/preset-env": "^7.9.5",
+    "babel-eslint": "^10.1.0",
     "babel-jest": "^25.4.0",
     "chai": "4.2.0",
     "chai-as-promised": "^7.1.1",
-    "eslint": "7.1.0",
-    "eslint-config-airbnb": "^18.1.0",
-    "eslint-config-airbnb-base": "14.1.0",
+    "eslint": "7.11.0",
```

Comment:
I accidentally committed these new updates to this branch but it's just minor package updates to fix security issues.